doc/book/box/atomic.rst

```diff
@@ -244,3 +244,27 @@ via telnet, via one of the connectors, or via the
 
 After a fiber has yielded and then has regained control, it immediately issues
 :ref:`testcancel <fiber-testcancel>`.
+
+.. atomic-transactional-manager
+
+--------------------------------------------------------------------------------
+Transactional manager
+--------------------------------------------------------------------------------
+
+Since version 2.6.1, Tarantool has another option for transaction behaviour that
+allows yielding inside a memtx transaction. This is controled by
+the *transactional manager*.
+
+The transactional manager is designed for isolation of concurrent transactions.
```

Comment:
I would add "and provides serializable transaction isolation level".

Reply:
added